docs/getting-started/README.md

```diff
@@ -0,0 +1,96 @@
+  - `kubectl apply -f ./docs/getting-started/pipeline.yaml`
+<!-- -->
+Our Triggers project will pickup from there:
+- We will setup an EventListener to accept and process GitHub Push events
+- A `TriggerTemplate` to create templated PipelineResource and PipelineRun resources per event received by the `EventListener`.
+<!-- -->
+- [Install the TriggerTemplate, TriggerBinding and EventListener](./triggers.yaml)
+  -  First, **edit** the `triggers.yaml` file to reflect the Docker repository you wish to push the image blob to.
+    - You will need to replace the `DOCKERREPO-REPLACEME` string everywhere it is needed.
+  - Once you have updated the triggers file, you can apply it!
+    - `kubectl apply -f ./docs/getting-started/triggers.yaml`
+  - If that succeeded, your cluster is ready to start handling Events.
+
+## Configure GitHub Webhook
+
+Now we need some events to work with. We need to configure GitHub to send events to our cluster, which is quick but has a few tricky parts.
+
+Because we are defining our EventListener's Spec as Public, we can expect to see a loadbalancer Service created. Do be aware this means that *your listener may be exposed to the internet*.
+
+- Establish your inbound EXTERNAL-URL hostname:
+  - In order to accept GitHub webhook payloads, our listener must be reachable from the internet.
+  - Our EventListener will create a [Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) for us, so we just need to determine the ingress hostname from the output of the Service. How the loadbalancer gets created will depend on how you are deploying this getting-started - AWS will behave differently than GCP, and so on.
+  - Inspect the output of `kubectl get service/getting-started-listener -n getting-started -o yaml` - there should be a hostname associated with the ingress now.
+  - This command should retrieve your hostname: `k get service/getting-started-listener -n getting-started -o json | jq ".status.loadBalancer.ingress[0].hostname"`
+
+Now configure GitHub webhooks to send *push* events to your clusters Loadbalancer ingress hostname.
+
+### GitHub Webhook Process
+
+Login to [GitHub](https://github.com) and configure your webhooks under the Repository settings. You can use the following images to see how this should look:
+
+![GitHub Webhook Push](images/trigger-webhook.png)
+
+Make sure to set `push` events as the type of events you want to send - push is the default.
+
+![GitHub Webhook Setup](images/trigger-webhook2.png)
+
+## Watch it work!
+
+- Commit and push an empty commit to your development repo:
+  - `git commit -a -m "build commit" --allow-empty && git push origin mybranch`
+- Now, you can follow the Task output in `kubectl logs`:
+  - First the image builder task:
+    - `kubectl logs -l somelabel=somekey --all-containers`
+  - Then our deployer task:
+    - `k logs -l tekton.dev/pipeline=getting-started-pipeline -n getting-started --all-containers`
```

Comment:
Can we `kubectl logs` instead of `k`? While it's quite common alias but some might not have that enabled.

Reply:
Yeah def. these `k` refs are all typos, ty